bin/alluxio-start.sh

```diff
@@ -260,13 +262,55 @@ start_worker() {
     exit 1
   fi
 
+  if [[ ! -z "${cache}" ]] ; then
+    if [[ ! -d "${cache}" ]] ; then
+      echo "Cache path ${cache} is not a directory; aborting"
```

Comment:
Does this error in the same way for the case of non-existent directory and if the specified path is a file? Would it be possible to separate those two cases?

Reply:
Sure I can make the error cases more granular